tests/admin/test_admin_users_endpoints.py

```diff
@@ -251,6 +252,30 @@ def test_get_user_noauth(client, db_session):
     r = client.get("/admin/user")
     assert r.status_code == 401
 
+# GET /list_policies test
+
+def test_list_policies(mock_arborist_requests, client, admin_user, encoded_admin_jwt):
+    mock_arborist_requests({"arborist/policy/": {"GET": ({"policy_ids": ["policy-abc", "policy-xyz"]}, 200)}})
```

Comment:
here is an example of how the data from arborist will be returned. Can you also add some tests testing what happens when either the correct parameter, incorrect parameter or no parameter is passed to the endpoint

#if exapnd = False
{
    "policies":
        [
            {
            "id":"test_admin",
            "description":"",
            "resource_paths":["/test_gateway"],
            "role_ids":["test_user"]
            }
        ]
}

#if expand = True
{
    "policies": [
            {
                "id":"test_admin",
                "description":"",
                "resource_paths":["/test_gateway"],
                "roles":[
                    {
                        "id":"test_user",
                        "description":"",
                        "permissions":[
                            {
                                "id":"test_access",
                                "description":"",
                                "action":{
                                    "service":"*",
                                    "method":"*"
                                },
                                "constraints":{

                                }
                            }
                        ]
                    }
                ]
            }
    ]
}